src/Crowdin.Api/Distributions/DistributionsApiExecutor.cs

```diff
@@ -52,6 +52,19 @@ public async Task<Distribution> AddDistribution(int projectId, AddDistributionRe
             return _jsonParser.ParseResponseObject<Distribution>(result.JsonObject);
         }
 
+        /// <summary>
+        /// Add distribution. Documentation:
+        /// <a href="https://support.crowdin.com/api/v2/#operation/api.projects.distributions.post">Crowdin API</a>
+        /// <a href="https://support.crowdin.com/enterprise/api/#operation/api.projects.distributions.post">Crowdin Enterprise API</a>
```

Comment:
It's a file-based documentation links

Reply:
For enterprise is there string-based documentation available ?

Reply:
Not yet